rudder-web/src/main/scala/com/normation/rudder/web/rest/RestExtractorService.scala

```diff
@@ -990,4 +957,63 @@ case class RestExtractorService (
 
   def extractId[T] (req : Req)(fun : String => Full[T])  = extractString("id")(req)(fun)
 
+  def extractNcfTechnique (json : JValue, methods: Map[BundleName, GenericMethod]) : Box[NcfTechnique] = {
```

Comment:
below things make me sad :(

Reply:
It's not that sad and was quite quick to do ... But yes it can be not done at all with other frameworks ...